### PR TITLE
added api-server custom ca docs (v2.9 / v2.13)

### PR DESCRIPTION
**Pull request by ozRunAI**

### CODE REVIEW

**Comment by ozRunAI** [comment]
@jasonnovichRunAI needs to be merged backwards to v2.9 as well please

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Modifying the API Server configuration differs between Kubernetes distributions:
 === "Native Kubernetes"
     * Locate the Kubernetes API Server configuration file. The file's location may differ between different Kubernetes distributions. The location for vanilla Kubernetes is `/etc/kubernetes/manifests/kube-apiserver.yaml`
     * Edit the document, under the `command` tag, add the __server__ configuration text from `Settings | General | Researcher Authentication`.   
+    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server, For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:41]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 2]
```diff
@@ -58,6 +59,8 @@ Modifying the API Server configuration differs between Kubernetes distributions:
             ...
             
     ```
+    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server, For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:62]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 3]
```diff
@@ -58,6 +59,8 @@ Modifying the API Server configuration differs between Kubernetes distributions:
             ...
             
     ```
+    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:62]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 4]
```diff
@@ -71,6 +74,8 @@ Modifying the API Server configuration differs between Kubernetes distributions:
     - "oidc-client-id=runai" # (1)
     ...
     ```
+    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server, For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:77]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 5]
```diff
@@ -190,6 +195,9 @@ Modifying the API Server configuration differs between Kubernetes distributions:
     --oidc-username-prefix=-
     ```
 
+    * For on-prem deployments, when Local Certificate Authority is configured - add the `oidc-ca-file` flag to the api server, For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:198]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}.
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 6]
```diff
@@ -38,6 +38,7 @@ Modifying the API Server configuration differs between Kubernetes distributions:
 === "Native Kubernetes"
     * Locate the Kubernetes API Server configuration file. The file's location may differ between different Kubernetes distributions. The location for vanilla Kubernetes is `/etc/kubernetes/manifests/kube-apiserver.yaml`
     * Edit the document, under the `command` tag, add the __server__ configuration text from `Settings | General | Researcher Authentication`.   
+    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:41]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}.
```

File: docs/admin/runai-setup/authentication/researcher-authentication.md [hunk 7]
```diff
@@ -71,6 +74,8 @@ Modifying the API Server configuration differs between Kubernetes distributions:
     - "oidc-client-id=runai" # (1)
     ...
     ```
+    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/researcher-authentication.md:77]:
```suggestion
    * For on-prem deployments, when Local Certificate Authority is configured, add the `oidc-ca-file` flag to the api server. For more information see [configuring the api server](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server){target=_blank}.
```

**Pull request merged**